packages/devtools-client-adapters/src/firefox/commands.js

```diff
@@ -113,46 +141,46 @@ function debuggeeCommand(script) {
   return Promise.resolve();
 }
 
-function navigate(url) {
+function navigate(url: string): Promise<*> {
```

Comment:
In the fx/types I created a `URL` type that was `export type URL = string` and I might try doing that here just because it looks better.  Not sure its really worth the effort.